space/bracketing.msl

```diff
@@ -24,24 +24,24 @@ license: MIT license (MIT) Copyright Huw Bowles May 2022
 // of directly taking the vector directly, take two samples of the texture
 // using coordinate frames at snapped angles, and then blend them based on
 // the angle of the original vector.
-void bracketing(float2 dir, out float2 vAxis0, out float2 vAxis1, out float blendAlpha) {
+void bracketing(float2 dir, float2 vAxis0, float2 vAxis1, float blendAlpha) {
     // Heading angle of the original vector field direction
-    float angle = atan(dir.y, dir.x) + TWO_PI;
+    float angle = atan2(dir.y, dir.x) + TWO_PI;
```

Comment:
Is `atan2()` equivalent to GLSL `atan()`?

Reply:
Confirmed, rendering `float3 color = float3(atan2(st.x, st.y)) * INV_PI + 0.5` in Metal and `vec3 color = vec3(atan(st.x, st.y)) * INV_PI + 0.5` in GLSL give the same result:

<img width="887" alt="image" src="https://github.com/user-attachments/assets/6d6b4651-323a-4d60-83f6-27136a3fd337" />